mava/configs/env/mabrax.yaml

```
@@ -15,5 +15,6 @@ eval_metric: episode_return
 
 kwargs:
   episode_length: 1000
-  homogenisation_method: max # Default is None. Options: [max, concat] if dimensions of observations and actions are not homogeneous across agents,
-  # "max" adds one-hot for the agent's ID. In this case, system.add_ids_to_agent_view should be set to False.
+  homogenisation_method: max # Default is None. Options: [max, concat] if dimensions of observations and actions are not homogeneous across agents.
+  # Note: When using "max", a one-shot vector representing the agent ID is added. Therefore ensure that the `system.add_ids_to_agent_view=False`
```

Sorry while we're here can we use the correct config name
```suggestion
  # Note: When using "max", a one-shot vector representing the agent ID is added. Therefore ensure that the `system.add_agent_ids=False`
```